Add ping endpoint

rasmusx opened the pull request:
Hello,

Here is MR for adding a "/_ping" endpoint that returns OK and maxrss.

Aims to solve issue: #232

kmike commented on splash/resources.py:
```diff
@@ -347,6 +347,18 @@ def render_POST(self, request):
+        request.setHeader(b"content-type", b"application/json")
+        return json.dumps({
+            "status": "ok",
+            "maxrss": resource.getrusage(resource.RUSAGE_SELF).ru_maxrss,
```
This has a problem - on OS X RUSAGE_SELF is in bytes, but on Linux it is in KBs. Check https://github.com/scrapinghub/splash/blob/3c9970ac96096a444ae9eb345d9670adc72bfbef/splash/utils.py#L114.

kmike commented:
Nice, thanks! Could you please add tests and docs?

rasmusx commented:
Thanks for pointing out my mistake.

I updated the code. Max RSS should be fixed now (always in bytes). I also updated docs and added a test case. Let me know if this correct/enough.

Thanks :)

kmike commented:
Thanks @rasmusx!
